src/views/registration-js/attendees/fields/birth.php

```diff
@@ -11,8 +11,9 @@
  * @see     Tribe__Tickets_Plus__Meta__Field__Birth
  */
 
-$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$required    = isset( $field->required ) && 'on' === $field->required;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 4 spaces

<sub>hash: e7c2c977c2ceea34fff7b220d220367c</sub>